src/make-http-request.coffee

```diff
@@ -44,12 +44,13 @@ makeHTTPRequest = (method, url, data, headers = {}, modify) ->
 
     req.end (error, response) ->
       delete getsInProgress[requestID]
+
       if error?.status is 408
         resolve makeHTTPRequest.apply null, originalArguments
-      else if error?
+      else if error? and response in error is false
```

Comment:
`in` here checks that an array includes an item. You probably want `of` (or `response not of error` in this case, [or `'response'` if you're checking the key, but I'm not really sure of the context]). Dumb CoffeeScript quirk.